src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -490,10 +506,10 @@ public static Matrix3x2 CreateTranslation(float xPosition, float yPosition)
             return result;
         }
 
-        /// <summary>Attempts to invert the given matrix. If the operation succeeds, the inverted matrix is stored in the result parameter.</summary>
-        /// <param name="matrix">The source matrix.</param>
-        /// <param name="result">The output matrix.</param>
-        /// <returns>True if the operation succeeded, False otherwise.</returns>
+        /// <summary>Inverts the specified matrix. The return value indicates whether the operation succeeded.</summary>
```

Comment:
Same here, I think `Attempts` or `Tries` first makes it more immediately obvious that this is
1. A badly named API that should have been called `TryInvert`
2. Is easier to read and ensures the user understand the operation/pattern